manifests/init.pp

```diff
@@ -175,24 +177,19 @@
     validate_re($logdir,'^(\$(?!/)|/).+')
     validate_re($rundir,'^(\$(?!/)|/).+')
 
+    if $ca_crl_pull_interval != 2 {
+      deprecation('pupmod::ca_crl_pull_interval', 'pupmod::ca_crl_pull_interval is depcrecated, the CRL cron job has been removed.')
```

Comment:
After re-looking at this, ca_crl_pull_interval should probably default to undef since it does nothing.